src/lib/src/auth/userinfo.service.ts

```diff
@@ -29,7 +30,7 @@ export class UserinfoService {
 
   getUserinfo(): Observable<Userinfo> {
     const url = `${this.authHost}/userinfo`;
-    const params = new HttpParams().set('scope', 'profile+apps');
+    const params = new HttpParams().set('scope', 'profile+apps+email');
```

Comment:
Hmm... any reason for adding email?  I think we usually display `firstname lastname`.  Not bad to have though.

Reply:
yeah, email is what is displayed in the dropdown on corporate. I saw that username was removed from the header in the latest designs, so I assumed it was to move that user info into the menu as was done on corporate. We were previously displaying `name` from the userinfo response. (Personally, I think email can get a little long and stretch the menu out wide and maybe name was a better choice.)